joblib/memory.py

```diff
@@ -464,18 +467,7 @@ def _cached_call(self, args, kwargs):
                                                    verbose=self._verbose)
         else:
             try:
-                t0 = time.time()
-                if self._verbose:
-                    msg = _format_load_msg(func_id, args_id,
-                                           timestamp=self.timestamp,
-                                           metadata=metadata)
-                out = self.store_backend.load_item([func_id, args_id], msg=msg,
-                                                   verbose=self._verbose)
-                if self._verbose > 4:
-                    t = time.time() - t0
-                    _, name = get_func_name(self.func)
-                    msg = '%s cache loaded - %s' % (name, format_time(t))
-                    print(max(0, (80 - len(msg))) * '_' + msg)
+                out = self.load(*args, **kwargs)
```

Comment:
This might hurt the performance here.
With this solution, we need to call `_get_output_identifier` twice and thus hash the argument multiple time. This can be slow in case where large argument are passed.